rebo/src/stdlib/mod.rs

```diff
@@ -399,3 +411,17 @@ fn file_read_to_string(name: String) -> Result<String, FileError> {
         }
     })()?
 }
+#[cfg(test)]
+mod test {
+    use crate::ReturnValue;
+    use crate::tests::test;
+
+    #[test]
+    fn test_float_rounding() {
+        test(r#"
+            let val = 3.14559;
+            let ceil_test = float::floor(val, 2);
+            print(ceil_test);
```

Comment:
To make sure you got the correct value, use `assert(1337 == b)`, which will make the test fail. Could you please test the `round` and `ceil` functions as well, and add more test numbers (one for each rounding-case)?

Reply:
Will do 👍🏼